scripts/macosx/build_qt5.sh

```diff
@@ -39,7 +39,7 @@ export QTDIR=$MIXXX_PREFIX/Qt-${VERSION_NUMBER}
 # Disable relocatable qt due to https://bugs.launchpad.net/mixxx/+bug/1871238
 # TODO(rryan): Disable framework build or use frameworks properly. Qt assumes
 # that if frameworks are enabled then they are used for deployment.
-./configure -opensource -prefix $QTDIR -sdk macosx${MIXXX_MACOSX_SDK} -system-sqlite -sql-sqlite -qt-zlib -platform macx-clang -release -confirm-license -securetransport -force-debug-info -no-feature-relocatable -nomake examples -nomake tests -skip qt3d -skip qtwebengine -I${MIXXX_PREFIX}/include -L${MIXXX_PREFIX}/lib
+./configure -opensource -prefix $QTDIR -sdk macosx${MIXXX_MACOSX_SDK} -ccache -system-sqlite -sql-sqlite -qt-zlib -platform macx-clang -release -confirm-license -securetransport -force-debug-info -nomake examples -nomake tests -skip qt3d -skip qtwebengine -I${MIXXX_PREFIX}/include -L${MIXXX_PREFIX}/lib
```

Comment:
Why do we put Qt in a separate subdirectory? That makes specifying the correct CMAKE_PREFIX_PATH much harder for Windows/macos devs. Can we change that?

Reply:
I don't know. I asked @rryan this on Zulip and did not get a response. AFAIK we could change it. But there is no pressing need to either.